thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/route.js

```diff
@@ -224,9 +225,15 @@ export default Ember.Route.extend({
             avgData[key].values = allValues;
           });
 
-          // Make custom calculations
-          avgData['responseRate'] = avgData['responseRate'] ? calculateRate(avgData['totalAlerts'], avgData['totalResponses']) : 'N/A';
-          avgData['precision'] = avgData['precision'] ? calculateRate(avgData['totalAlerts'], avgData['trueAnomalies']) : 'N/A';
+          // Gather totals and make custom calculations
+          let pTrue = getTotalValue('trueAnomalies.tot');
+          let pNew = getTotalValue('newTrend.tot');
```

Comment:
can't we use `const` here if we don't expect the variable to change?

Reply:
for this instance, we do expect it to change, as its within a `availableGroups.forEach` loop.